lib/api/bucketDeleteTagging.js

```diff
@@ -0,0 +1,52 @@
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const { pushMetric } = require('../utapi/utilities');
+const metadata = require('../metadata/wrapper');
+const util = require('node:util');
+
+/**
+ * Bucket Delete Tagging - Delete a bucket's Tagging
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+async function bucketDeleteTagging(authInfo, request, log, callback) {
```

Comment:
I don't think we need a callback with async, you can just return the value as we need to await the function anyway.

Reply:
Thanks and I agree, but removing the callback now will require code [changes in Arsenal](https://github.com/scality/Arsenal/blob/development/8.1/lib/s3routes/routes/routeDELETE.ts#L13), where cloudserver API handlers are called, with a callback... How about creating a ticket to switch the entire project from callbacks to async/await?

Reply:
If that is the case I wouldn't use async at all and just use the callback, saving the switch to async/await for the new work. I would also make sure that the "switch entire project" is split up a bit as that would be a massive PR. We also probably need to sync multiple projects and teams together, similar to the node 16 work.

Reply:
Thanks, but now I have a question: you suggested these changes in Will's PR [here](https://github.com/scality/cloudserver/pull/4481#discussion_r891495697) on 7.10 branch, do you think it's better to revert these changes for now and use callbacks? if yes, I will do that tomorrow :smile:

Reply:
No problem! Done here: https://github.com/scality/cloudserver/pull/4588/commits/4c6075708660c9fbe54c8b44ff888e8318188707